spec/plugins_manager_spec.coffee

```diff
@@ -89,14 +89,14 @@ describe 'PluginsManager', ->
             @subject.notify('addOrder', { order_id: 1 })
 
             expect(window.sa_plugins.analytics_plugin)
-              .to.deep.equal({ order_id: 1, shop_code: @session.shop_code, analytics_session: @session.analytics_session, configuration: @plugin.configuration })
+              .to.deep.equal({ order_id: 1, shop_code: @session.shop_code, analytics_session: @session.analytics_session, configuration: @plugin.configuration, data: @plugin.data })
```

Comment:
[Minor] Add a dummy `data: { attribute: 'attribute' }` in the mocked `@plugins_response`